arrows/ceres/bundle_adjust.cxx

```diff
@@ -92,17 +92,8 @@ class bundle_adjust::priv
   {
   }
 
-  priv(const priv& other)
-  : solver_options(other),
-    camera_options(other),
-    verbose(other.verbose),
-    loss_function_type(other.loss_function_type),
-    loss_function_scale(other.loss_function_scale),
-    ceres_callback(),
-    m_logger( vital::get_logger( "arrows.ceres.bundle_adjust" ))
-  {
-  }
-
+  /// the Ceres solver options
+  ::ceres::Solver::Options options;
```

Comment:
This "options" member can go away because it is now provided by inheriting from the "solver_options" helper class.